main_loop.py

```diff
@@ -101,7 +101,7 @@ def main_loop():
     feed_watch_dog()
 
     # Increment loopcount
-    patterns.loopcount += 1
+    patterns.loopcount = (patterns.loopcount + 1) % 16
```

Comment:
⭐ ⭐ I don't think this should happen here, what if there's a pattern that wants to have a longer than 16 phase animation?